_internal/src/types.ts

```diff
@@ -310,14 +310,17 @@ export type MutatorCallback<Data = any> = (
   currentData?: Data
 ) => Promise<undefined | Data> | undefined | Data
 
-export type MutatorOptions<Data = any> = {
+export type MutatorOptions<SWRData = any, Data = SWRData> = {
```

Comment:
```suggestion
export type MutatorOptions<Data = any, MutationData = Data> = {
```

Maybe this naming is better? I think we’d like to have a JSDoc here explaining what these generics mean actually. Still kinda confusing.

Reply:
Updated as your suggestion, JSDoc added as well.